ocp_resources/virtual_machine_clone.py

```diff
@@ -0,0 +1,87 @@
+from ocp_resources.constants import TIMEOUT_4MINUTES
+from ocp_resources.resource import NamespacedResource
+from ocp_resources.utils import TimeoutSampler
+from ocp_resources.virtual_machine import VirtualMachine
+
+
+class VirtualMachineClone(NamespacedResource):
+    api_group = NamespacedResource.ApiGroup.CLONE_KUBEVIRT_IO
+
+    def __init__(
+        self,
+        name=None,
+        namespace=None,
+        source_name=None,
+        source_type=None,
+        target_name=None,
+        label_filters=None,
+        annotation_filters=None,
+        new_mac_addresses=None,
+        new_smbios_serial=None,
+        yaml_file=None,
+        **kwargs,
+    ):
+        """
+        Create VirtualMachineClone object.
+
+        Args:
+            name (str): VirtualMachineClone name
+            source_name (str): the clone's source name
+            source_type (str): the clone's source type
+            target_name (str): the clone's target name
+            label_filters (list, optional): List of label filters, e.g. ["*", "!someKey/*"]
+            annotation_filters (list, optional): List of annotation filters, e.g. ["firstKey/*", "secondKey/*"]
+            new_mac_addresses (dict, optional): Dict of new MAC addresses, {interface_name: mac_address}
+            new_smbios_serial (str, optional): the clone's new smbios serial
+        """
+        super().__init__(
+            name=name,
+            namespace=namespace,
```

Comment:
Name, namespace and yaml_file should be passed as kwargs as well

Reply:
done